conductr_cli/sandbox_init.py

```diff
@@ -1,62 +1,102 @@
-from conductr_cli import docker_machine, terminal
+from conductr_cli import docker, docker_machine, terminal
 import logging
+import re
 from subprocess import CalledProcessError
 
 
 def init(args):
     """`sandbox init` command"""
 
     log = logging.getLogger(__name__)
-    vm_name = docker_machine.vm_name()
+
+    try:
```

Comment:
Why we don't do check which VM the user is using and based on that perform different init operations?